src/shop/actions/discount/get-discounts.action.ts

```diff
@@ -4,6 +4,13 @@ import { IDiscount, IDiscountFilter } from '../../types';
 import { Pagination } from '../../../common/types/response';
 import { DiscountErr, errorFactory } from '../../errors/discount/errors';
 
+/**
+ * Retrieves a paginated list of discounts based on provided filters.
+ *
+ * @param {Fetcher} fetcher - The fetcher instance used for making API requests.
+ * @param {IDiscountFilter} [filters] - Optional filters to apply when fetching discounts.
```

Comment:
```suggestion
 * @param {IDiscountFilter} filters - Optional filters to apply when fetching discounts.
```

Est-ce que les crochets ("[ ]") sont nécessaires dans le commentaire ?